geoportal/c2cgeoportal_geoportal/__init__.py

```diff
@@ -493,8 +493,11 @@ def call_hook(settings, name, *args, **kwargs):
 
 
 def notfound(request):
-    del request
-    return HTTPNotFound()
+    return {
```

Comment:
you can use: `HTTPNotFound(request.path_info)`
see https://docs.pylonsproject.org/projects/pyramid/en/latest/api/httpexceptions.html#pyramid.httpexceptions.HTTPNotFound

Reply:
No because I will get the same message as with c2cwsgiutils, here: https://github.com/camptocamp/c2cwsgiutils/blob/master/c2cwsgiutils/errors.py#L80

Reply:
ok, I get it now !